src/plugins/uk.ac.kcl.mdeoptimise.ui/src/uk/ac/kcl/ui/classpath/MDEOClasspathContainerPage.xtend

```diff
@@ -10,19 +10,19 @@ import org.eclipse.swt.SWT
 import org.eclipse.swt.widgets.Composite
 import org.eclipse.swt.widgets.Label
 
-class MDEOptimiserClassPathContainerPage implements IClasspathContainerPage {
+class MDEOClasspathContainerPage implements IClasspathContainerPage {
 	
 	override finish() { true }
 	
 	override getSelection() {
-		JavaCore.newContainerEntry(MDEOClassPathContainerInitializer.MDEO_LIBRARY_PATH);
+		JavaCore.newContainerEntry(new MDEOClasspathContainer().path);

```

Comment:
Shouldn't this also have `"/dsl"` appended?

Reply:
That is correct, it gets that value from the getPath method.

Reply:
Of course. That makes sense.